smart-frontend/app/src/api/httpClient/axiosBasedHttpClient.ts

```diff
@@ -29,9 +29,10 @@ export class AxiosBasedHttpClient implements HttpClient {
       withCredentials: true,
       headers: {
         'Content-Type': 'application/json',
+        'X-Requested-With': 'XMLHttpRequest',
       },
-      xsrfCookieName: 'csrftoken',
-      xsrfHeaderName: 'X-CSRFToken',
+      // xsrfCookieName: 'csrftoken',
```

Comment:
please, add comments, when we should uncomment that code, otherwise, please, remove